docs/fast_data/tutorials/low-code.mdx

```diff
@@ -26,348 +26,342 @@ To do that, in the console, you will:
-Its content must be replaced with the following:
+### Link to Microservice
 
-```json {4,6,10} showLineNumbers
-{
-  "version": "1.0.0",
-  "config": {
-    "registry": {
-      "outgoing": {
-        "order": {
-          "conditions": {
-            "registry_to_order": {
-              "condition": {
-                "registry_id": "registry_id"
-              },
-              "oneToMany": true
-            }
-          }
-        }
-      },
-      "order": {
-        "outgoing": {
-          "registry": {
-            "conditions": {
-              "order_to_registry": {
-                "condition": {
-                  "registry_id": "registry_id"
-                },
-                "oneToMany": false
-              }
-            }
-          }
-        }
-      }
-    }
-  }
-}
-```
+Before receiving messages from the ingestion topic defined in the projection, you have to link it to a SoR microservice. 
 
-In this example, we can see that the relationship goes from registry to order (from line 4 to line 16) and vice versa.
+Go back to the `Services` section of the `delivery` SoR, and select the microservice you have attached during the previous paragraph: there, on the `Projections` card, you can pick one or more projections.
 
-The relationship is represented by the following two conditions:
-- _registry_id_ (from order): _registry_id_ (from the updated registry). The condition is _One to One_, meaning that one order is related to one and only one registry item.
-- _registry_id_ (from registry): _registry_id_ (from the updated order). The condition is _One to Many_, meaning that one registry might be related to multiple orders.
+![Link Projection to SoR Microservice](img/35-link-pr-to-PS.png)
 
-The condition is used to generate a `where` condition inside a MongoDB query: in this case, the condition can be translated as
-"where the registry_id inside order projection equals the value of the registry_id field of the registry projection".
+In this particular case, both `registry` and `orders` will be linked to the same microservice.
 
-In the context of the Real-Time Updater, the only relationship needed is the one going _from order to registry_, because when an order is updated, we want to see if there is already a document inside
-the _registry_ Projections that shares the same _registry_id_. If so, that document will be used as a reference to update the related Single View.
+Going back to the projection's page, you can see that the microservice linked to it will appear under the `Microservice` card.
 
-But in the example above, we decided to configure the relationship both ways to be able to use the same JSON also in the service that aggregates the Single View (the _Single View Creator_).
-</TabItem>
+### Specifications
 
+Here you can find the specifications about the projections we are going to use in this tutorial. 
 
-</Tabs>
+<details>
+  <summary> Registry </summary>
+  <p>
 
-#### Projection Changes Schema
+#### Fields
```

Comment:
I'd move these fields list below the sentence: 'Now we need to define the fields that we want to store in the Projection. You can do that by clicking the `Add field` button located at the top-right of the `Fields card`.'